src/app/components/applications/SSV/MyAccount/components/Withdraw/components/ValidatorFlow/ValidatorFlow.tsx

```diff
@@ -74,18 +75,17 @@ const ValidatorFlow = () => {
   const withdrawSsv = async () => {
     setIsLoading(true);
     const success = await ssvStore.withdrawSsv(withdrawValue.toString());
-    const response = await Validator.getInstance().clusterByHash(clusterStore.getClusterHash(cluster.operators));
+    const response = await Validator.getInstance().clusterByHash(getClusterHash(cluster.operators, walletStore.accountAddress));
     const newCluster = response.cluster;
-    newCluster.validatorCount = newCluster.validatorCount;
     newCluster.operators = cluster.operators;
     processStore.setProcess({
       processName: 'single_cluster',
       // @ts-ignore
```

Comment:
This will likely cause a major bug. Remove the @ts-ignore and handle the required change